### PR TITLE
Feat/guidelines ds

### PR DESCRIPTION
**Pull request by samuel-gomez-axa**
- some icones size fixed
- add guidelines UX
- add documentation

### CODE REVIEW

**Review by guillaume-chervet — DISMISSED** [review]
Nice job Samuel ! Awesome

**Comment by gcruchon** [comment]
Wow, that's a huge PR and a lot of work! Thanks very much!

Just noticed that the following lines are still in the package.json, but I think they are useless (l. 18 & 19):
```js
    "css": "npm -C ./storybook/styles run css",
    "css:build": "npm -C ./storybook/styles run css:build",
```

Maybe you can remove them as well?

**Comment by samuel-gomez-axa** [comment]
Sure, you're right @gcruchon , I propose to do this in another PR because this subject of this PR, ok ?

**Comment by gcruchon** [comment]
No problem. Let's add an issue, then? I can contribute :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,4 @@
-# Contributing to @axa-fr/react-toolkit
-
-First, ensure you have Node.js < 12 and the [latest `npm`](https://docs.npmjs.com/).
+## Installation
```
[the remark this answers is not in the document]
Comment (samuel-gomez-axa) [reply CONTRIBUTING.md:1]: i just wanted to delete the node version requirement. i didn't want to delete the title. i'll fixed it, thank you.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -56,39 +42,28 @@ Into a terminal, to start to develop a component you can run
...
 ```
 
-After all this, if your development affect css, you can run
-
-```sh
-npx gulp all
-```
-
-to regenerate css
+After all this, if your development affect css, you can run `npx gulp all` to regenerate css
```
[the remark this answers is not in the document]
Comment (samuel-gomez-axa) [reply CONTRIBUTING.md:63]: yes good idea

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (samuel-gomez-axa) [reply CONTRIBUTING.md:3]: done in last commit

File: CONTRIBUTING.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (samuel-gomez-axa) [reply CONTRIBUTING.md]: done in last commit

File: CONTRIBUTING.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gcruchon) [reply CONTRIBUTING.md:3]: 👍

**Review by guillaume-chervet — APPROVED** [review]

**Comment by gcruchon** [comment]
Hello,

Looks like there are some broken dependencies in non committed pug files.
E.g. : [/storybook/design-system/src/components/form/mixins/mixin-form-field-card.pug](https://github.com/AxaGuilDEv/react-toolkit/blob/feat/guidelines-ds/storybook/design-system/src/components/form/mixins/mixin-form-field-card.pug), line 7:
```yaml
          include ../../../pages/style/icons/templates/ok.pug
```

This template has moved to `../../../pages/guidelines/icones/templates/ok.pug`

WIth a search in files on `/style/icons/templates/` you should find all of them (100+ occurences in 48 files)

BTW, the french `icones` lacks a bit of consistancy here :)

**Review by arnaudforaison — APPROVED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -10,29 +8,22 @@ cd react-toolkit
...
 If you want to use these projects, you must:
 
-```sh
-cd examples/demo
-npm i
-```
-
 ```sh
 cd storybook/design-system
 npm i
```
[the remark this answers is not in the document]
Comment (arnaudforaison) [reply CONTRIBUTING.md]: I think "ci" it's more appropriated

**Pull request merged**